t/server/controller/bad_request.t

```diff
@@ -0,0 +1,55 @@
+use strict;
+use warnings;
+use lib 't/lib';
+
+use YAML::XS;
+use Cpanel::JSON::XS ();
+use MetaCPAN::Server ();
+use MetaCPAN::TestHelpers;
```

Comment:
I don't see where this is being used. We should explicitly import and functions if it is being used.

Reply:
I simplified the module includes a bit.